docker/package/tezos_setup_wizard.py

```diff
@@ -382,12 +382,18 @@ def import_baker_key(self):
 
             key_mode_query = get_key_mode_query(key_import_modes)
 
-            self.import_key(key_mode_query, "Baking")
+            not_set_up = True
+            while not_set_up:
+                self.import_key(key_mode_query, "Baking")
 
-            proc_call(
-                f"sudo -u tezos {suppress_warning_text} tezos-client {tezos_client_options} "
-                f"setup ledger to bake for {baker_alias} --main-hwm {self.get_current_head_level()}"
-            )
+                try:
+                    proc_call(
+                        f"sudo -u tezos {suppress_warning_text} tezos-client {tezos_client_options} "
+                        f"setup ledger to bake for {baker_alias} --main-hwm {self.get_current_head_level()}"
+                    )
+                    not_set_up = False
+                except KeyboardInterrupt:
```

Comment:
This doesn't seem to work
```
Please open the Tezos Baking app on your ledger or
press Ctrl+C to go back to the key import mode selection.
Select a key to import from the ledger.
You can choose one of the suggested derivations or provide your own:

'ledger://live-fossa-eager-walrus/':

1. bip25519/0h/0h,  address: tz1evTDcDb1Da5z9reoNRjx5ZXoPXS3D1K1A, balance: 37774.881223 ꜩ
2. ed25519/0h/0h,   address: tz1faCC6Fm9gorxDaa2TUmQsYXeCJyv6rFGJ, balance: 0 ꜩ
3. secp256k1/0h/0h, address: tz2TzKnNpGKdGr5roWhNBHhYJZULAASaGFn9, balance: 0 ꜩ
4. P-256/0h/0h,     address: tz3bfris46EYvYgPfCsiAWoRBv7D2kGzSj3i, balance: 0 ꜩ

5. Specify derivation path
6. Go back
> 1
Please validate (and write down) the public key hash displayed on the Ledger,
it should be equal
to `tz1evTDcDb1Da5z9reoNRjx5ZXoPXS3D1K1A`:
Tezos address added: tz1evTDcDb1Da5z9reoNRjx5ZXoPXS3D1K1A
Setting up the ledger:
* Main chain ID: NetXnHfVqm9iesp -> NetXLH1uAxK7CCh
* Main chain High Watermark: 582672 (round: 0) -> 149241 (round: 0)
* Test chain High Watermark: 520586 (round: 0) -> 520586 (round: 0)
^CAuthorized baking for address: tz1evTDcDb1Da5z9reoNRjx5ZXoPXS3D1K1A
Corresponding full public key: edpkv9HH7Goq7YNyRfiYN4UY8etHU3wGnbEaBAnLPoY6QU53279PPh
Error in Tezos Setup Wizard, exiting.
The error has been logged to /media/serokell/pyrometer/build/tezos_setup_wizard.log
$ cat /media/serokell/pyrometer/build/tezos_setup_wizard.log
[Errno 1] Operation not permitted
```
My impression is that the error that is thrown is not a `KeyboardInterrupt`, but a `CalledProcessError` produced by the `proc_call` above

Reply:
Actually maybe I failed to reprodude the exact issue

Reply:
At what moment `Ctrl + C` is fired? During the ledger waiting you to permit the operation or the moment exact after it?

Reply:
The error does not occur on 22.04, that was the problem

Reply:
> During the ledger waiting you to permit the operation or the moment exact after it?

After the ledger starts showing `Review Request` for `setup ledger to bake` command

Reply:
Hmm, I've rebuilt and reinstalled the package, and now I cannot reproduce it as well :see_no_evil:

Reply:
False alarm then

Reply:
I reproduced it on 20.04